maliput/src/utility/mod.rs

```diff
@@ -53,18 +53,27 @@ pub type ObjFeatures = maliput_sys::utility::ffi::Features;
 pub fn generate_obj_file(
     road_network: &RoadNetwork,
     dirpath: impl AsRef<Path>,
-    fileroot: impl AsRef<Path>,
+    fileroot: &String,
```

Comment:
Have we consider using `impl AsRef<str>`? This would allow us to pass either `&str` or `String`, and get a reference to them.

Reply:
Nice catch!
Done